libid/fractals.cpp

```diff
@@ -669,6 +733,95 @@ void mandel_perturb(
     delta_n.real(dnr);
 }
 
+double diff_abs(const double c, const double d)
```

Comment:
declare this static and move it to burning_ship.cpp

Reply:
Done